bundles/tampere/content-editor/view/SideContentEditor.js

```diff
@@ -497,22 +647,70 @@ Oskari.clazz.define('Oskari.tampere.bundle.content-editor.view.SideContentEditor
          * @param  {Boolean}    deleteFeature is delete feature
          */
         sendRequest: function (requestData, deleteFeature) {
+            var overlay = Oskari.clazz.create('Oskari.userinterface.component.Overlay');
+            overlay.overlay('body');
+            overlay.followResizing(true);
+            var spinner = Oskari.clazz.create('Oskari.userinterface.component.ProgressSpinner');
+            spinner.insertTo('body');
+            spinner.start();
+
             var me = this,
                 okButton = Oskari.clazz.create('Oskari.userinterface.component.Button'),
                 url = null,
                 wfsLayerPlugin = me.sandbox.findRegisteredModuleInstance('MainMapModule').getPluginInstances('WfsLayerPlugin');
 
             okButton.setTitle(me.loc.buttons.ok);
-            if (me.operationMode === 'create') {
+            var multipleFeaturesData = [];
+
+            if (me.operationMode === 'create' && !me.editMultipleFeatures) {
                 url = Oskari.urls.getRoute('InsertFeature');
-            } else {
+            } else if(!me.editMultipleFeatures) {
+                url = Oskari.urls.getRoute('SaveFeature');
+            } else if(me.editMultipleFeatures) {
                 url = Oskari.urls.getRoute('SaveFeature');
+                var elements = jQuery('.getinforesult_table');
+                var index = 0;
+                jQuery.each(elements || [], function(k1, v1) {
+                    //Skip first element since we have that already in requestData.
+                    if(index > 0) {
+                        //Copy requestData object for each of the selected features.
+                        var tempFeature = me._cloneObject(requestData[0]);
+                        var tempFeatureFields = [];
+                        var tds = jQuery(v1).children('tr').children('td');
+                        //Go through all the tds to assign tempFeature the correct values.
```

Comment:
Gathering form data here as well? Could use the same function as before if it's refactored as one.